assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -5075,4 +5075,14 @@ public String intToTime(int time) {
     public String getContentReviewServiceName() {
         return this.contentReviewService.getServiceName();
     }
+    
+    @Override
+    public String getUserAssignmentModify(String modifier) {
+    	try {
+    		return userDirectoryService.getUser(modifier).getDisplayName();
+    	}catch (UserNotDefinedException e) {
+    		return resourceLoader.getString("user.modify.unknown", "");
```

Comment:
```suggestion
    		return resourceLoader.getString("user.modify.unknown");
```

Is the extra param necessary?

Reply:
yes, if the properties do not exist, it takes the default value ""